sections/04_study.md

```diff
@@ -602,7 +602,7 @@ study of micro-scale ecosystems within and around us. In recent years, machine
 learning has proved to be a powerful tool for metagenomic analysis. 16S rRNA has
 long been used to deconvolve mixtures of microbial genomes, yet this ignores
 more than 99% of the genomic content. Subsequent tools aimed to classify
-300-3000 base pair reads from complex mixtures of microbial genomes based on
+300 bp-3000 bp reads from complex mixtures of microbial genomes based on
```

Comment:
Has `bp` been defined somewhere previously?  I didn't find it in a quick search of the HTML.